CMakeLists.txt

```diff
@@ -57,6 +57,27 @@ if (MATERIALX_BUILD_GEN_MDL)
     set(MATERIALX_INSTALL_MDL_MODULE_PATH ${MATERIALX_INSTALL_STDLIB_PATH} CACHE FILEPATH "Install path for mdl module.")
 endif()
 
+###############################################################################
```

Comment:
For consistency with other CMake scripts in MaterialX, I would recommend removing these hash separators, and letting the text comments stand on their own.